Src/ParachutesControl.c

```diff
@@ -8,7 +8,23 @@
 #include "FlightPhase.h"
 #include "Data.h"
 
-static int MONITOR_FOR_PARACHUTES_PERIOD = 1000;
+// Pressure at spaceport america in 100*millibars on May 27, 2018
+static const int SEA_LEVEL_PRESSURE = 101421.93903699999; //TODO: THIS NEEDS TO BE UPDATED AND RECORDED ON LAUNCH DAY
+static const int MAIN_DEPLOYMENT_ALTITUDE = 4572; // Units in meters. Equivalent of 15000 ft.
```

Comment:
Hey is this 15000 ft above ground level or sea level? It should be above ground level.

Reply:
Is it bad if we just hard code this? It's the easiest and most reliable option at the moment.